src/ReefGuideAPI.jl

```diff
@@ -55,79 +55,93 @@ Load regional data to act as an in-memory cache.
 OrderedDict of `RegionalCriteria` for each region.
 """
 function setup_regional_data(config::Dict)
+    reef_data_path = config["prepped_data"]["PREPPED_DATA_DIR"]
+    reg_cache_dir = config["server_config"]["REGIONAL_CACHE_DIR"]
+    reg_cache_fn = joinpath(reg_cache_dir, "regional_cache.dat")
+
     if @isdefined(REGIONAL_DATA)
         @debug "Using previously generated regional data store."
         sleep(1)  # Pause so message is noticeably visible
-        return REGIONAL_DATA
```

Comment:
I think you still want the `return REGIONAL_DATA`, right?
Also, I think remove the `sleep`

Reply:
I like sleep but fair enough...